spanner/api/Spanner.Samples.Tests/InstanceConfigPropertiesTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Spanner.Admin.Instance.V1;
+using Xunit;
+
+namespace Spanner.Samples.Tests
+{
+    public class InstanceConfigPropertiesTest
```

Comment:
We don't need these tests here, this is a given from Protobuf. You can assume this will always work.